core/server/data/schema/schema.js

```diff
@@ -172,6 +172,7 @@ module.exports = {
         slug: {type: 'string', maxlength: 150, nullable: false, unique: true},
         secret: {type: 'string', maxlength: 150, nullable: false},
         redirection_uri: {type: 'string', maxlength: 2000, nullable: true},
+        blog_uri: {type: 'string', maxlength: 2000, nullable: true},
```

Comment:
Hmm I've added the `blog_uri` to the schema to be able to track a change.
So if config.url changes, how do we know?Maybe i am blind to see there is an easier solution 😬